Tweeter/Scripts/app.js

```diff
@@ -0,0 +1,24 @@
+﻿$("#register-username").focusout(function () {
+    // alert("defocused!!!"); this is correct so far
+    console.log($(this).val());
+   // $(this).val();//this refers to #register-username; this refers to current element you are sitting at
+    $.ajax({
+        url: "/api/TwitUsername?candidate=" + $(this).val(),
+
+        method: 'GET'
+    }).success(function (response) {
+        console.log(response);
+        if(response.exists){
+            $("username-ans").addClass("glyphicon-ok");
+        } else {
+            $("username-ans").addClass("glyphicon-remove");
```

Comment:
when toggling two classes, be sure to remove the one you don't want to in the right situation (e.g. if we're adding `glyphicon-ok`, be sure to call `removeClass` on the `glyphicon-remove` class)